lasagna-master/ena/lasagna_master.go

```diff
@@ -0,0 +1,37 @@
+package lasagna
+
+func PreparationTime(layers []string, timePerLayer int) int {
+	const averageTimePerLayer = 2
+	if timePerLayer == 0 {
+		timePerLayer = averageTimePerLayer
+	}
+	return len(layers) * timePerLayer
+}
+
+func Quantities(layers []string) (int, float64) {
+	const noodlesPerLayer = 50 // grams
+	const saucePerLayer = 0.2  // liters
+	noodlesLayers := 0
+	sauceLayers := 0
```

Comment:
Cleaner as a one-liner as they are related.

Reply:
You mean `var noodlesLayers, sauceLayers int`? Not sure though what you mean they are related?

Reply:
`noodlesLayers, sauceLayers := 0, 0.0`

Reply:
ah, ok, using `var` to declare and rely on the default values is nice as well IMO

Reply:
Yes it is, but in this case is better to do it my way IMO as you would have an int and a float64 and you wouldn't need the cast in the return.

Reply:
Got it, you prefer performance than art ... :-)

Reply:
Come on, my favorite language is Python. That's impossible!